src/QueryRenderer.ts

```diff
@@ -8,17 +8,16 @@ import { DateFallback } from './DateFallback';
 import { getTaskLineAndFile, replaceTaskWithTasks } from './File';
 
 import type { IQuery } from './IQuery';
+import { explainResults, getQueryForQueryRenderer } from './lib/QueryRendererHelper';
+import type { GroupDisplayHeading } from './Query/GroupDisplayHeading';
+import type { QueryResult } from './Query/QueryResult';
+import type { TaskGroups } from './Query/TaskGroups';
 import {
     type HappensDate,
     createPostponedTask,
-    explainResults,
     getDateFieldToPostpone,
-    getQueryForQueryRenderer,
-} from './lib/QueryRendererHelper';
-import type { GroupDisplayHeading } from './Query/GroupDisplayHeading';
-import type { QueryResult } from './Query/QueryResult';
-import type { TaskGroups } from './Query/TaskGroups';
-import { shouldShowPostponeButton } from './Scripting/Postponer';
+    shouldShowPostponeButton,
+} from './Scripting/Postponer';
```

Comment:
Do you understand why the order of imports has changed here again? Isn't this the one we had to fight with when merging the 'postponement' PR?

Reply:
This is due to new Postponer.ts file but generally speaking I choose to don’t spend my time on this and just accept the weirdness of JetBrains as long as the thing is working

Reply:
> This is due to new Postponer.ts file but generally speaking I choose to don’t spend my time on this and just accept the weirdness of JetBrains as long as the thing is working

Fine. Very happy not to fix it in this PR - but after the Postpone feature is out, I would like to try and stop this happening in future:

- #2456

Reply:
That would be great! Thanks!